File: 프로그래머스/lv0/120883. 로그인 성공？/로그인 성공？.js
function solution(id_pw, db) {
  var answer = '';
  for (const dbItem of db) {
    if (id_pw[0] === dbItem[0]) {
      if (id_pw[1] === dbItem[1]) {
        return "login"
      } else {
        return "wrong pw"
      }
    }
  }
  return "fail";
}

File: 프로그래머스/lv0/120883. 로그인 성공？/로그인 성공？.py
def solution(id_pw, db):
    answer = ''
    id, password = id_pw[0], id_pw[1]
    
    for check in db:
        if id == check[0]:
            if password == check[1]:
                return "login"
            if password != check[1]:
                return "wrong pw"
    return "fail"